### PR TITLE
[now-ruby] Add test fixture for ActiveSupport gem

### PR DESCRIPTION
**Pull request by m5o**
* add ruby ActiveSupport test fixture
  > ActiveSupport is a Ruby module & a stand-alone component of Rails that includes additional functionality to core Ruby classes, like Array, String & Numeric.
  :link: https://guides.rubyonrails.org/active_support_core_extensions.html

* add `Date.current + 10.years` calculation example

### CODE REVIEW

File: packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    zeitwerk (2.2.2)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  activesupport (~> 6.0)
+
+BUNDLED WITH
+   1.17.3
```
Comment (styfle) [inline packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock]: Does it make a difference which bundle version is used? Can you upgrade to 2.x.x? Most other tests are using `2.0.1`.

File: packages/now-ruby/test/fixtures/09-activesupport/Gemfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem 'activesupport', '~> 6.0'
```
Comment (styfle) [inline packages/now-ruby/test/fixtures/09-activesupport/Gemfile]:
```suggestion
gem "activesupport", "~> 6.0"
```

Our other test fixtures use double quotes here.

File: packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock (hunk 1 above)
Comment (m5o) [reply packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock]: I've installed ruby `v2.5.7` which is used in production by ZEIT and this is the out-of-the-box `bundler` version. But no problem will bundle with the latest `bundler` version 👍

Comment (styfle) [reply packages/now-ruby/test/fixtures/09-activesupport/Gemfile.lock]: Interesting, I think I'm confusing the static ruby builds with the serverless functions. The static ruby is 2.6 right now and the latest bundler.

**Review by styfle — APPROVED** [review]

**Pull request merged**